backend/attendance-service/src/main/java/org/egov/enrichment/AttendanceLogEnrichment.java

```diff
@@ -35,8 +38,9 @@ public void enrichAttendanceLogCreateRequest(AttendanceLogRequest attendanceLogR
 
     public void enrichAttendanceLogUpdateRequest(AttendanceLogRequest attendanceLogRequest) {
         List<AttendanceLog> attendanceLog = attendanceLogRequest.getAttendance();
-        AuditDetails auditDetails = getAuditDetails(attendanceLogRequest);
+        String byUser = attendanceLogRequest.getRequestInfo().getUserInfo().getUuid();
         attendanceLog.forEach(e -> {
```

Comment:
Instead of the `forEach` construct, I suggest the `for (Object o : array)` is an easier-to-read construct. 
Could you please replace all forEach with the for loop.

Reply:
I have replaced the **forEach** construct with  **for (Object o : array)** in whole code base.